pocean/dsg/trajectoryProfile/cr.py

```diff
@@ -180,7 +180,7 @@ def to_dataframe(self, clean_cols=True, clean_rows=True):
         for i in np.arange(profile_indexes.size):
             ei = si + o_index_var[i]
             p[si:ei] = profile_indexes[i]
-            r[si:ei] = traj_indexes[r_index_var[i]]
+            r[si:ei] = np.array(traj_indexes[r_index_var[i]])
```

Comment:
@kwilcox I don't like this solution but I do not have a Windows machine to investigate this and figure out a better one.

It seems that, on Windows , `traj_indexes[r_index_var[i]]` is a `np.str_` instead of a `np.array`. Casting it back to array works, otherwise we get garbage in the masked `r` array. See https://ci.appveyor.com/project/ocefpaf/pocean-core/build/1.0.3/job/gnm13r4d2vm0koqi#L953